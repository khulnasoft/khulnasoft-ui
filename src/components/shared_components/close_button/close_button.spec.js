import { shallowMount } from '@vue/test-utils';
import CloseButton from './close_button.vue';

describe('Clear Icon Button', () => {
  let wrapper;

  const defaultPropsData = {
    label: 'Dismiss',
  };

  const createComponent = (propsData = defaultPropsData) => {
    wrapper = shallowMount(CloseButton, { propsData });
  };

  beforeEach(() => {
    createComponent();
  });

  it('renders successfully', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
