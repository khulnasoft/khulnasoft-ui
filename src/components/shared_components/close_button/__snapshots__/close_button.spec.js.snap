// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clear Icon Button renders successfully 1`] = `
<gl-button-stub
  aria-label="Dismiss"
  buttontextclasses=""
  category="tertiary"
  icon="close"
  size="small"
  variant="default"
/>
`;
